run/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from storage.models import File
+from game_runner import settings
+import uuid
+
+
+# Initial scheme
+# FilePath
+#   +file : File
+#   +path : String
+#   +run : Run
+#
+# Run
+#   +token : Token
+#   +request_time : DateTime
+#   +start_time : DateTime
+#   +end_time : DateTime
+#   +input_file_paths : FilePath[]
+#   +output_file_paths : FilePath[]
+#   +log : File
+#   +game : Game
+#
+# API Methods:
+#   +run((game : Game, file_paths : FilePath[])[]) : tokens String[]
+#   +report(from_time : DateTime) : runs : Run[]
+
+class FilePath(models.Model):
+    file = models.ForeignKey(File)
+    # definition = models.ForeignKey('FileDefinition') later should be uncommented
+    run = models.ForeignKey('Run')
```

Comment:
Is this ForeignKey needed when we have a Many-To-Many relation with Run?
Django provides `FilePath.run_set` when we have Many-To-Many with Run.